src/include/rados/librados.hpp

```diff
@@ -1019,8 +1019,6 @@ namespace librados
     void set_assert_version(uint64_t ver);
     void set_assert_src_version(const std::string& o, uint64_t ver);
 
-    const std::string& get_pool_name() const;
-
```

Comment:
I was afraid about linking problems too, so added the _const_ modifier to _std::string get_pool_name()_ -- please refer line 626. C++ does not respect the returned type during method overloading, I suppose.